views/partials/what_to_do.hbs

```diff
@@ -2,22 +2,30 @@
   <li>What to Do</li>
   <li>
     <span class="numeral">1</span>
-    <span class="bold">Change your password</span> on these sites and anywhere else you've used the same password.
+    <p>
+      <span class="bold">Change your password</span> on these sites and anywhere else you've used the same password.
+    </p>
   </li>
   <li>
     <span class="numeral">2</span>
-    Make your answers to <span class="bold">security questions</span> just as strong as your passwords.
+    <p>
+      Make your answers to <span class="bold">security questions</span> just as strong as your passwords.
+    </p>
   </li>
   <li>
     <span class="numeral">3</span>
-    <span class="bold">Use password managers</span> like 1Password, LastPass, or Dashlane to generate strong passwords for you, remember them for you, and fill them into websites so you don't have to type them in.
+    <p>
+      <span class="bold">Use password managers</span> like 1Password, LastPass, or Dashlane to generate strong passwords for you, remember them for you, and fill them into websites so you don't have to type them in.
+    </p>
   </li>
   <li>
     <span class="numeral">4</span>
-    <span class="bold">Subscribe</span> for alerts from Firefox Monitor to learn sooner about your compromised accounts.
+    <p>
+      <span class="bold">Subscribe</span> for alerts from Firefox Monitor to learn sooner about your compromised accounts.
+    </p>
   </li>
   <li>
-    <span class="scan-another-email">Scan another email address.</span> 
+    <label class="scan-another-email">Scan another email address.</label> 
```

Comment:
a11y: This `<label>` should probably be linked to something (like an `<input>`).